crates/zoon/src/style/font.rs

```diff
@@ -65,36 +145,95 @@ impl<'a> Font<'a> {
         self
     }
 
+    /// Set the font line height which is the height between lines of text in an
+    /// element.
+    /// # Example
+    /// ```no_run
+    /// use zoon::*;
+    ///
+    /// let paragraph = Paragraph::new()
+    /// .s(Font::new()
+    /// .line_height(150))
+    /// .content("Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...");
+    /// ```
     pub fn line_height(mut self, line_height: u32) -> Self {
         self.static_css_props.insert("line-height", px(line_height));
         self
     }
 
+    /// Set the font as Italic.
+    /// # Example
+    /// ```no_run
+    /// use zoon::*;
+    ///
+    /// let paragraph = Paragraph::new()
+    /// .s(Font::new()
+    /// .italic())
+    /// .content("Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...");
+    /// ```
     pub fn italic(mut self) -> Self {
         self.static_css_props.insert("font-style", "italic");
         self
     }
 
+    /// Collapse white space and suppress line breaks.
+    /// # Example
+    /// ```no_run
+    /// use zoon::*;
+    ///
+    /// let paragraph = Paragraph::new()
+    /// .s(Font::new()
+    /// .no_wrap())
+    /// .content("Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...");
+    /// ```
     pub fn no_wrap(mut self) -> Self {
         self.static_css_props.insert("white-space", "nowrap");
         self
     }
 
     pub fn wrap_anywhere(mut self) -> Self {
```

Comment:
Missing doc comment.

Reply:
Yes !!! so True I thought about it and then I forgot :D , I am going to add it once I pulled all the new commits.

Reply:
I updated